lib/galaxy/webapps/tool_shed/framework/middleware/hg.py

```diff
@@ -75,6 +75,9 @@ def __call__( self, environ, start_response ):
                     connection.execute( sql_cmd )
                     connection.close()
         elif cmd in [ 'unbundle', 'pushkey' ]:
+            if self.config[ 'disable_push' ]:
```

Comment:
`s/self.config[ 'disable_push' ]/self.config.disable_push/`

Should fix the traceback in https://jenkins.galaxyproject.org/job/docker-toolshed/1994/testReport/junit/tool_shed.functional.test_0310_hg_push_from_api/TestHgWebFeatures/test_0010_edit_and_commit/ :
```
File "/galaxy/lib/galaxy/webapps/tool_shed/framework/middleware/hg.py", line 78, in __call__
    if self.config[ 'disable_push' ]:
KeyError: 'disable_push'
```